app/src/components/features/rules/RulesListContainer/index.js

```diff
@@ -231,7 +232,7 @@ const RulesListContainer = ({ isTableLoading = false }) => {
   return (
     <>
       {/* Page content */}
-
+      {user.details.isLimitReached ? <FeatureLimiterBanner /> : null}
```

Comment:
Add a check here for user is logged in or not, might break for users have rules in logged out state